source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/ReportDashboard.cshtml

```diff
@@ -16,50 +16,69 @@
-                <!-- /ko -->
-            </tr>
-        </thead>
-        <tbody data-bind="foreach: items">
-            <tr>
-                <td data-bind="text: period"></td>
-                <td data-bind="text: fullName"></td>
-                <td data-bind="text: currentPosition"></td>
-                <td data-bind="text: seniority"></td>
-                <td data-bind="text: responsibleId"></td>
-                <td><span class="state-doc" data-bind="text: stateName, attr: { 'class': stateClasses }"></span></td>
-                <td><a class="link-responsable top-tip" data-bind="if: state() == 6, attr: {href: calificationUrl}" data-tips="Ver Evaluación"><i class="icon view"></i><span>Ver Evaluación</span></a></td>
-            </tr>
-        </tbody>
-    </table>
-    <script type="text/javascript">
-        var employeePeriod = "@ViewBag.Period";
-    </script>
+  </div>
+  <table class="table">
+    <thead>
+      <tr data-bind="foreach: headers">
+        <!-- ko if: sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
+          <i class="icon arrow"></i>
+          <span data-bind="text: title"></span>
+        </th>
+        <!-- /ko -->
+        <!-- ko if: !sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }"><span data-bind="text: title"></span></th>
+        <!-- /ko -->
+      </tr>
+    </thead>
+    <tbody data-bind="foreach: items">
+      <tr>
+        <td data-bind="text: period"></td>
+        <td data-bind="text: fullName"></td>
+        <td data-bind="text: currentPosition"></td>
+        <td data-bind="text: seniority"></td>
+        <td class="responsible-column">
+          <span data-bind="if: isEvaluationManager">
+            <a class="calification-links edit top-tip" data-bind="click: showResponsibleManager">
+              <i class="icon edit"></i>
+              <span data-bind="text: responsibleId()"></span>
+            </a>
+          </span>
+          <!-- ko ifnot: isEvaluationManager -->
+          </span><span data-bind="text: responsibleId"></span>
+          <!-- /ko -->
+        <td><span class="state-doc" data-bind="text: stateName, attr: { 'class': stateClasses }"></span></td>
+        <td><a class="link-responsable top-tip" data-bind="if: state() == 6, attr: {href: calificationUrl}" data-tips="Ver Evaluación"><i class="icon view"></i><span>Ver Evaluación</span></a></td>
+      </tr>
+    </tbody>
+  </table>
+  <script type="text/javascript">
+      var employeePeriod = "@ViewBag.Period";
+  </script>
 </div>
 
+<div class="content-modal clearfix">
+  <div class="modal-title">
+    <h4>Cambiar Responsable</h4>
+  </div>
+  <div class="search-container">
+    <input type="text" class="search" autocomplete="off" data-bind="value: responsibleManagerModel.newResponsible, event: {keypress: responsibleManagerModel.onEnter}" />
```

Comment:
Yes, I think `submit` data binding on a form would be better, but take care that form has his own restrictions, for example they cannot be nested. So, if it is working now, do not change it hurriedly .